500 internal error on status event

<!-- 
- describe the problem you've encountered and the expected behavior
- screenshots of branch protection settings and Kodiak messages/behavior are helpful
-->
Traceback:

```
AssertionError: null
  File "starlette/exceptions.py", line 73, in __call__
    raise exc from None
  File "starlette/exceptions.py", line 62, in __call__
    await self.app(scope, receive, sender)
  File "starlette/routing.py", line 590, in __call__
    await route(scope, receive, send)
  File "starlette/routing.py", line 208, in __call__
    await self.app(scope, receive, send)
  File "starlette/routing.py", line 41, in app
    response = await func(request)
  File "fastapi/routing.py", line 133, in app
    raw_response = await dependant.call(**values)
  File "kodiak/main.py", line 96, in webhook_event
    queue=redis_webhook_queue, event_name=github_event, payload=event
  File "kodiak/queue.py", line 279, in handle_webhook_event
    await status_event(queue, StatusEvent.parse_obj(payload))
  File "kodiak/queue.py", line 144, in status_event
    pr_results = await asyncio.gather(*pr_requests)
  File "kodiak/queries/__init__.py", line 1020, in get_open_pull_requests
    session=self.session, installation_id=self.installation_id
  File "kodiak/queries/__init__.py", line 1288, in get_headers
    session=session, installation_id=installation_id
  File "kodiak/queries/__init__.py", line 1278, in get_token_for_install
    assert res.status_code < 300
```

I cannot see res as it is filterred, I checked permission and it's all match . This is a self-hosted instance for github enterprise. 

related issue: https://github.com/chdsbd/kodiak/issues/562

--- a/bot/kodiak/queries/__init__.py
+++ b/bot/kodiak/queries/__init__.py
@@ -1247,7 +1247,7 @@ def generate_jwt(*, private_key: str, app_identifier: str) -> str:
     This is different from authenticating as an installation
     """
     issued_at = int(datetime.now().timestamp())
-    expiration = int((datetime.now() + timedelta(minutes=10)).timestamp())
+    expiration = int((datetime.now() + timedelta(minutes=9, seconds=30)).timestamp())
     payload = dict(iat=issued_at, exp=expiration, iss=app_identifier)
     return jwt.encode(payload=payload, key=private_key, algorithm="RS256").decode()
 

--- a/bot/kodiak/queries/__init__.py
+++ b/bot/kodiak/queries/__init__.py
@@ -1277,7 +1277,8 @@ async def get_token_for_install(
                 Authorization=f"Bearer {app_token}",
             ),
         )
-    assert res.status_code < 300
+    if res.status_code > 300:
+        raise Exception(f"Failed to get token, github response: {res.text}")
     token_response = TokenResponse(**res.json())
     installation_cache[installation_id] = token_response
     return token_response.token

